docs/started-resources.md

```diff
@@ -1,52 +1,52 @@
-- [Cicero UI](https://github.com/accordproject/cicero-ui): a library of react components for visualizing, creating and editing Accord Project templates
-
-## Developers Resources
-
-All the Accord Project technology is being developed as open source. The software packages are being actively maintained on [GitHub](https://github.com/accordproject) and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
-
-Join us on the [#technology-wg Slack channel](https://accord-project-slack-signup.herokuapp.com) for technical discussions and weekly updates.
-
-### Cicero
-
-- GitHub: https://github.com/accordproject/cicero
-- [Cicero Slack Channel](https://accord-project.slack.com/messages/CA08NAHQS/details/)
-- Technical Questions and Answers on [Stack Overflow](https://stackoverflow.com/questions/tagged/cicero)
-
-### Ergo
-
-- GitHub: https://github.com/accordproject/ergo
-- The [Ergo Language Guide](logic-ergo) is a good place to get started with Ergo.
-- [Ergo Slack Channel](https://accord-project.slack.com/messages/C9HLJHREG/details/)
+---
+id: started-resources
+title: Resources
+---
+
+## Accord Project Resources
+
+- The Main Web site includes latest news, links to working groups, organizational announcements, etc. : https://www.accordproject.org
+- This Technical Documentation: https://docs.accordproject.org
+- Recording of Working Group discussions, Tutorial Videos are available on Vimeo: https://vimeo.com/accordproject
+- Join the [Accord Project Slack](https://accord-project-slack-signup.herokuapp.com) to get involved!
+
+## User Content
+
+Accord Project also maintains libraries containing open source, community-contributed content to help you when authoring your own templates:
+
+- [Model Repository](https://models.accordproject.org/) : a repository of open source Concerto data models for use in templates
+- [Template Library](https://templates.accordproject.org/) : a library of open source Clause and Contract templates for various legal domains (supply-chain, loans, intellectual property, etc.)
+
+## Ecosystem & Tools
+
+Accord Project is also developing tools to help with authoring, testing and running accord project templates.
+
+### Editors
+
+- [Template Studio](https://studio.accordproject.org/): a Web-based editor for Accord Project templates
+- [VSCode Extension](https://marketplace.visualstudio.com/items?itemName=accordproject.cicero-vscode-extension): an Accord Project extension to the popular [Visual Studio Code](https://visualstudio.microsoft.com/) Editor
+- [Emacs Mode](https://github.com/accordproject/ergo/tree/master/ergo.emacs): Emacs Major mode for Ergo (alpha)
+- [VIM Plugin](https://github.com/accordproject/ergo/tree/master/ergo.vim): VIM plugin for Ergo (alpha)
+
+### User Interface Components
+
+- [Markdown Editor](https://github.com/accordproject/markdown-editor): a general purpose react component for markdown rendering and editing

```

Comment:
As the Markdown Editor was moved to `web-components` monorepo you may update that link to https://github.com/accordproject/web-components/tree/master/packages/ui-markdown-editor